Makefile

```diff
@@ -75,27 +80,10 @@ build/aux: build/locator build/marketplace
 
 build: build/insomnia build/aux
 
-install/miner: build/miner
+install: all
```

Comment:
So I think that the `install` target must require the `build` target, not the `all` one.

Reply:
Why?

Reply:
install is usually separated because it oftens require higher privilegies, but it's not this case and we don't need building and installing division, so I'd personally either depend install on all or vice versa

Reply:
@antmat +1